static/js/bot_data.js

```diff
@@ -35,12 +24,13 @@ var bot_data = (function () {
     exports.update = function bot_data__update(email, bot_update) {
         var bot = bots[email];
         _.extend(bot, _.pick(bot_update, bot_fields));
-        set_can_admin(bot);
         send_change_event();
     };
 
-    exports.get_editable = function bots_data__get_editable() {
-        return _.filter(bots, function (bot) { return bot.can_admin; });
+    exports.get_all = function bots_data__get_all() {
```

Comment:
You can remove `bots_data_get_all()` here.  We no longer use that coding convention, as most browsers give us decent tracebacks without the long noisy name.